### DISCUSSION TITLE
Is there a way to get the current locale for the page you're editing in preview?

### DISCUSSION THREAD

**Discussion by richardvanbergen** [payloadcms/payload#173] [Q&A] [answered]
Hey guys, pretty simple question I need to know which locale the user is currently editing when creating the preview button. Just wondering if there's an officially supported way of doing this?

**Comment by jmikrut** [comment, marked as answer]
Hey @richardvanbergen — we can pass in the `locale` to that function for sure. Give us a bit today and we'll deploy a new version with locale added to the `generatePreviewURL` function.

Good call!

**Reply by richardvanbergen** [reply]
Wonderful that would certainly be a lot less hacky than the way I'm doing it now 😆
